src/ui/components/results/resultsheadercomponent.js

```diff
@@ -86,6 +86,20 @@ export default class ResultsHeaderComponent extends Component {
     });
   }
 
+  /**
+   * Returns the currently applied nlp filter nodes, with nlp filter nodes that
+   * are duplicates of other static filter nodes removed.
```

Comment:
"duplicates of other filter nodes removed"? They won't be duplicates of static filters, necessarily.

Reply:
updated!